### DISCUSSION TITLE
[@types/express]: Change Request body type from Record<string, any> to Record<string, unknown>

### DISCUSSION THREAD

**Discussion by bbennett7** [DefinitelyTyped/DefinitelyTyped#53754] [Issues with a @types package]
Right now, the `any` type forces typecasting with `as` if you don't want to get stuck with `any`. If this is changed to `unknown`, we can run our own type checks and enforce types through these rather than just casting. This will also help avoid the eslint error `@typescript-eslint/no-unsafe-assignment` without ignoring individual lines.

**Comment by darcyrush** [comment]
@bbennett7 did you find a way to solve this?

I have been trying variations of the following with no luck

```ts
declare global {
  namespace Express {
    interface Request extends Record<string, unknown> {
      body: unknown;
    }
  }
}
```

**Reply by darcyrush** [reply]
I used local type definitions originally, but a global declaration merge would cut down on the local type overrides considerably
